### ISSUE TITLE
Create auto-apply label policies KQL for pre-defined managed property

### ISSUE THREAD

**Issue by Breaker119** [MicrosoftDocs/microsoft-365-docs#7350]
I've had a support case open on this topic for almost a month now with no resolution so I thought I'd try here. 

I have mapped a crawled property (ows_PromotedState) to a RefinableString as outlined in the documentation (I've also tried RefinableInt00) and the KQL search works as expected in SharePoint (e.g. RefinableString110=2) - all published news posts are returned. However, when building the "Apply label to content matching this query" or while testing via Content Search the pre-defined managed property is not recognized and both the label policy and Content Search will error.

Is there a different KQL syntax that is supposed to be used that is not shown in the documentation? Are pre-defined managed properties available for use in the KQL search query for Auto-applied label policies (and Content Search) as shown here?
https://docs.microsoft.com/en-us/microsoft-365/compliance/auto-apply-retention-labels-scenario?view=o365-worldwide#create-auto-apply-label-policies

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 38185ae9-6017-736a-adf2-1a6f62d4ae09
* Version Independent ID: 1f57cbcc-bdea-e712-2f35-d9282c133433
* Content: [Use retention labels to manage the lifecycle of documents stored in SharePoint - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/auto-apply-retention-labels-scenario?view=o365-worldwide&source=docs)
* Content Source: [microsoft-365/compliance/auto-apply-retention-labels-scenario.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/auto-apply-retention-labels-scenario.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
@Breaker119 Yes, per https://docs.microsoft.com/en-us/microsoft-365/compliance/apply-retention-labels-automatically?view=o365-worldwide#auto-apply-labels-to-content-with-keywords-or-searchable-properties: " .. you can use mappings at the tenant level with the predefined managed properties that are enabled as refiners by default (RefinableDate00-19, RefinableString00-99, RefinableInt00-49, RefinableDecimals00-09, and RefinableDouble00-09). The example you gave was "RefinableString110=2", and this is outside the supported range for auto-apply labels (RefinableString00-99). We can't provide technical support with GitHub Issues, but I wanted to make sure you were aware of these supported ranges.

**Comment by Breaker119** [comment]
Ah - yes...very astute observation on 110! However, I started my experiment out by using RefineableInt01 and then moved on to RefinableString when that didn't work. I've now tried again with RefinableInt10 and unfortunately get Potential error: "Unknown property name: RefinableInt10=" in red text

When pushing through, though, and ignoring the error, I do appear to get the correct results. More testing will confirm!

I appreciate your response and I will pass your comment on to the support engineer - thank you.
